examples/flutter_gallery/lib/demo/animation/home.dart

```diff
@@ -119,8 +121,10 @@ class _SliverAppBarDelegate extends SliverPersistentHeaderDelegate {
   final double maxHeight;
   final Widget child;
 
-  @override double get minExtent => minHeight;
-  @override double get maxExtent => math.max(maxHeight, minHeight);
+  @override
+  double get minExtent => minHeight;
+  @override
+  double get maxExtent => math.max(maxHeight, minHeight);
```

Comment:
this is a similar one to the edge case i mentioned earlier... i guess the new formatting is fine...